frontend/src/views/MovieAddScreen.vue

```diff
@@ -107,21 +107,21 @@ export default {
         };
 
         const { data: movie } = await axios.post(
-          `${import.meta.env.VITE_API_URL}/api/cinema/movies`,
+          `${import.meta.env.VITE_API_URL}/api/cinema/movies/`,
           {
             title: this.title,
             duration: Number(this.duration),
             description: this.description,
             actors: this.selectedActorIds,
-            genres: this.selectedGenreIds
+            genres: this.selectedGenreIds,
           },
           movieConfig
         );
 
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies/${movie.id}/upload-image/`, data, imageConfig);
```

Comment:
Use `VITE_API_URL` to obtain the images in this call.

Reply:
It didn't work